module/metrics/transaction_validation.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/onflow/flow-go/module"
+)
+
+type NamespaceType string
+
+// TransactionValidationCollector the metrics for transaction validation functionality
+type TransactionValidationCollector struct {
+	transactionValidated         prometheus.Counter
+	transactionValidationSkipped prometheus.Counter
+	transactionValidationFailed  *prometheus.CounterVec
+}
+
+// interface check
+var _ module.TransactionValidationMetrics = (*TransactionValidationCollector)(nil)
+
+// NewTransactionValidationCollector creates new instance of TransactionValidationCollector
+func NewTransactionValidationCollector() *TransactionValidationCollector {
+	return &TransactionValidationCollector{
+		transactionValidated: promauto.NewCounter(prometheus.CounterOpts{
+			Name:      "transaction_validation_successes_total",
+			Namespace: namespaceAccess,
+			Subsystem: subsystemTransactionValidation,
+			Help:      "counter for the validated transactions",
+		}),
+		transactionValidationSkipped: promauto.NewCounter(prometheus.CounterOpts{
+			Name:      "transaction_validation_skipped",
```

Comment:
circling back to Jordan's comment about the counter best practice. let's add total
```suggestion
			Name:      "transaction_validation_skipped_total",
```